packages/bsky/src/services/indexing/plugins/post.ts

```diff
@@ -96,6 +106,18 @@ const insertFn = async (
     return null // Post already indexed
   }
 
+  if (obj.reply) {
```

Comment:
could we do this in the initial insert? no biggie if not

Reply:
We totally can—my thinking was that it should be rare enough that we need to perform the write that it shouldn't matter a ton either way.

Reply:
actually yeah that's a good point - seems fine 👍